walrus-tests/tests/spec-tests.rs

```diff
@@ -115,8 +115,21 @@ fn run(wast: &Path) -> Result<(), failure::Error> {
         .args(extra_args)
         .output()
         .context("executing `spectest-interp`")?;
+
+    // If the interpreter exits with success it may still have failed some
+    // tests. Check the output for `X/Y tests passed.` and make sure `X` equals
+    // `Y`.
     if output.status.success() {
```

Comment:
Seems like maybe we should file a bug upstream to exit with code = number of failures or something.

Reply:
https://github.com/WebAssembly/wabt/issues/1002